server/src/main/java/com/coincoinche/events/GameStartedEvent.java

```diff
@@ -0,0 +1,14 @@
+package com.coincoinche.events;
+
+public class GameStartedEvent extends Event {
+  private String gameId;
+
+  public GameStartedEvent(String gameId) {
+    super(EventType.GAME_STARTED);
+    this.gameId = gameId;
+  }
+
+  public String getGameId() {
```

Comment:
Spring understands getters and setters directly: this can be serialised to JSON without adding any decorator, the corresponding field will be `gameId` 😍

Reply:
Nice!